app/cafe.py

```diff
@@ -0,0 +1,25 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):

```

Comment:
Code Efficiency: Instead of using `dict.get(key)` to check if a key exists in a dictionary, you can directly use `if key in dict`. This is more Pythonic and efficient. For example, you can rewrite this line as `if 'vaccine' not in visitor:`.